readthedocs/proxito/tests/test_full.py

```diff
@@ -213,6 +213,7 @@ def test_project_nginx_serving_unicode_filename(self):
 @override_settings(
     PYTHON_MEDIA=False,
     PUBLIC_DOMAIN='readthedocs.io',
+    RTD_BUILD_MEDIA_STORAGE='readthedocs.rtd_tests.storage.BuildMediaFileSystemStorageTest',
```

Comment:
Only this test was expecting to use `readthedocs.rtd_tests.storage.BuildMediaFileSystemStorageTest` instead of `readthedocs.proxito.tests.storage.BuildMediaStorageTest`, couldn't figure out how to make it work with the other one